### DISCUSSION TITLE
Fast ebooks for KOReader

### DISCUSSION THREAD

**Discussion by Kollerov** [koreader/koreader#10413] [Q&A]
Hi everyone!

I am trying to find the best affordable and fast e-book for KOReader app. I have Pocketbook 623, but it is too slow. Furthermore, I am grateful that it's possible to customize almost any PDF file with KOReader, especially The Economist magazine, it's game changer for me. 

Thank you in advance for your feedback! ^_^

**Comment by NiLuJe** [comment]
A Kindle PW5 if you can get one able to be jailbroken (which is not going to be easy); or a Kobo Sage.
